apps/demos/menuMeta.json

```diff
@@ -3313,6 +3313,24 @@
       }
     ]
   },
+  {
```

Comment:
Let's place it between HtmlEditor and Diagram

Reply:
2679e0c3362d769e1ea3315578e64523e07689d8